### PR TITLE
chore: added json and mdx files to add several repos to the video category

### PR DESCRIPTION
**Pull request by adwetzelnr**
chore:  added chore: added json and mdx files to add several repos to the video category

Associated with [Issue #790 ](https://github.com/newrelic/opensource-website/issues/790)

### CODE REVIEW

**Comment by jpvajda** [comment]
@zstix this is a rather large PR so I added to in review for our team to look at.

**Review by roadlittledawn — DISMISSED** [review]

File: src/data/project-main-content/newrelic-video-anvato-js.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+path: "/projects/newrelic/video-akamai-js"
+date: "05/01/2020"
+title: "New Relic JW Player JS Tracker"
+projectConfig: "src/data/projects/newrelic-video-akamai-js.json"
```
[the remark this answers is not in the document]
Comment (adwetzelnr) [reply src/data/project-main-content/newrelic-video-anvato-js.mdx:5]: Looks like I did munge some of the file names and content.  What is the best way to resolve

[the remark this answers is not in the document]
Comment (roadlittledawn) [reply src/data/project-main-content/newrelic-video-anvato-js.mdx:5]: not sure i quite follow. can you make the changes in your forked branch and push it up?

[the remark this answers is not in the document]
Comment (adwetzelnr) [reply src/data/project-main-content/newrelic-video-anvato-js.mdx]: pushed additional changes to address naming error in Anvato and Akamai mdx files.  Updated all mdx to remove commented text. Changed the 'description' in a few of the json files.

hope this is good to go

File: src/data/project-main-content/newrelic-video-anvato-js.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+consectetur adipiscing elit.
+
+> [name="Leslie Webb"] Vitae enim egestas egestas at gravida arcu, amet in. Facilisis at
+massa amet, aliquet dui semper. Sit placerat sed et ornare faucibus
+egestas sit nisl, diam.
+
+> [name="Bildad the Shuhite"] Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis
+vestibulum. Maecenas faucibus mollis interdum. Maecenas sed diam
+eget risus varius blandit sit amet non magna.
+-->
```
[the remark this answers is not in the document]
Comment (adwetzelnr) [reply src/data/project-main-content/newrelic-video-anvato-js.mdx]: removed comments from all video-related mdx files

**Review by jpvajda — APPROVED** [review]
This checks out code looks good as does the PR preview

**Pull request merged**